src/component/common/util.js

```diff
@@ -6,5 +6,25 @@ const dateTimeOptions = {
     minute: '2-digit',
     second: '2-digit',
 };
+// todo for a more comprehensive list use of moment.tz from https://github.com/moment/moment-timezone
+const predefinedLocale = [
+    {
+        locale: 'nb-NO',
+        timezone: 'UTC',
+    },
+    {
+        locale: 'us-US',
+        timezone: 'America/New_York',
+    },
+    {
+        locale: 'en-GB',
+        timezone: 'Europe/London',
+    },
+];
+export const formatFullDateTimeWithLocale = (v, locale) => {
+    let found = predefinedLocale.find(v => v.locale === locale);
+    dateTimeOptions.timeZone = found ? found.timezone : 'UTC';
```

Comment:
This actually limits the possible time-zones to the defined list. By omitting the timezone we will get the current timezone in the browser, which is probably closer to what the user expects. 

By only changing the locale, and not the timezone, we allow the user to change the formatting. What do you think?